src/scss/themes/_small.scss

```diff
@@ -20,6 +20,11 @@
 		@include oGridRespondTo($until: M) { // Mobile screens only
 			display: none;
 		}
+
+		//hack because IE11 and flexbox aren't friends
+		.top-stories__freeform-twocolumns & {
```

Comment:
This class looks specific to the next codebase. Is there a way we can have the fix but not be tied to Next's classes?